framework/src/modules/chain/blocks/verify.js

```diff
@@ -40,9 +40,8 @@ const verifyPreviousBlockId = (block, lastBlock, genesisBlock) => {
 };
 
 class BlocksVerify {
-	constructor({ storage, exceptions, slots, dposModule, genesisBlock }) {
+	constructor({ storage, exceptions, slots, genesisBlock }) {
```

Comment:
Do we have a test for verifying the attributes the constructor expects?

Reply:
Not that I'm aware of.